main.py

```diff
@@ -1,127 +1,201 @@
 #!/usr/bin/env python
-import sys
+
+import argparse
+import logging
 import os
+import signal
+import sys
+import threading
+import traceback
+
+from logging.handlers import RotatingFileHandler
 from subprocess import call
 from time import sleep
-import argparse
-#Welcome to pyLCI innards
-#Here, things are about i and o, which are input and output
-#And we output things for debugging, so o goes first.
+
+from apps.manager import AppManager
+from helpers import read_config
+from input import input
 from output import output
+from splash import splash
```

Comment:
Importing splash file might also result in an exception, but it's better if that doesn't stop pyLCI loading - on ZeroPhone, pyLCI is pretty much *the* ZeroPhone UI and thus has to disregard errors in sections as non-critical as a splashscreen module (however, logging the import error would be great, of course).